android/fastlane/Appfile

```diff
@@ -0,0 +1,2 @@
+json_key_file("../google-services.json") # Path to the json secret file - Follow https://docs.fastlane.tools/actions/supply/#setup to get one
+package_name("com.blacktaxandwhitebenefits") # e.g. com.krausefx.app
```

Comment:
Isn't this supposed to have gfapp?

Reply:
![image](https://github.com/user-attachments/assets/d0cd6fd4-f593-444c-bfc8-171bfbfd7ff6)

This is in the prod firebase project details FYI

Reply:
I will probably just delete the Fastlane stuff for this PR